### PR TITLE
Fix repo-list private and total count bugs

### PR DESCRIPTION
**Pull request by zeripath**
Fixes issue with the repo-list total count number not showing up and just sets the "private" toggle to match repo.is_private state.

A more complete refactor of these toggles is required in future but this is a simple backportable bug-fix

Signed-off-by: Andrew Thornton <art27@cantab.net>

### CODE REVIEW

**Comment by zeripath** [comment]
(Fixes bugs reported by @CirnoT)

**Comment by CirnoT** [comment]
Public repos of limited/private organizations still don't show as either Private or Public, only in undefined state.

**Comment by zeripath** [comment]
Sigh ... I've set it so that "private" shows the limited and private org public repos.

**Comment by CirnoT** [comment]
I don't think I agree with them being classified as private, considering they are currently shown as public (both via icon as well as color)

**Comment by 6543** [comment]
this is kind/breaking for the API !!!

**Comment by 6543** [comment]
OnlyPrivate -> AndPrivate  or am I'm wrong?

**Comment by zeripath** [comment]
@CirnoT changing it to do the obvious thing would break the API as @6543 says - and therefore unbackportable.

**Comment by zeripath** [comment]
but of course this still doesn't completely work...

pub repo of pub org seems not be shown.

**Comment by zeripath** [comment]
WHY IS THIS SUCH A MESS

**Review by 6543 — APPROVED** [review]

**Comment by lafriks** [comment]
Something wrong with swagger, duplicate description

File: templates/swagger/v1_json.tmpl [hunk 1]
```diff
@@ -1772,7 +1772,7 @@
           {
             "type": "boolean",
             "description": "only include private repositories this user has access to (defaults to false)",
```
Comment (6543) [inline templates/swagger/v1_json.tmpl]:
```suggestion
            "description": "show only pubic, private or all repositories (defaults to all)",
```

Comment (6543) [reply templates/swagger/v1_json.tmpl]: this shuld fix lint too ...

**Review by lafriks — APPROVED** [review]

**Review by CirnoT — APPROVED** [review]

**Review by guillep2k — APPROVED** [review]

**Comment by guillep2k** [comment]
Please send backport

**Pull request merged**